third_party/color_emoji/png.py

```diff
@@ -17,7 +17,12 @@
 # Google Author(s): Behdad Esfahbod
 #
 
-import struct, StringIO
+import struct
+import sys
+if sys.version_info >= (3,0,0): # Python3
+   from io import StringIO
```

Comment:
no, please use `io.BytesIO` for both python2 and 3. The data in question is binary data (i.e. `bytes`), not textual data (`str`). Later in the file, you don't have to decode bytes as "iso-8859-1" just to write them to a text stream  `io.StringIO`. Just use BytesIO for both 2 and 3.